recipes/configs/llama2/1B_full_ppo.yaml

```diff
@@ -0,0 +1,182 @@
+# Config for single device RLHF full finetuning using PPO in ppo_full_finetune_single_device.py
```

Comment:
This is the only 1B recipe in the repo, but I think this recipe in particular benefits from starting out with smaller models. The reward model I've linked here has been trained using TRL and the HF page includes the relevant commands to reproduce the model, so hopefully users should be able to adapt the pipeline to their own tasks.